definitions/npm/autobind-decorator_v2.x.x/flow_v0.25.x-/autobind-decorator_v2.x.x.js

```diff
@@ -0,0 +1,20 @@
+declare module 'autobind-decorator' {
+    declare export interface TypedPropertyDescriptor<T> {
+      enumerable?: boolean;
+      configurable?: boolean;
+      writable?: boolean;
+      value?: T;
+      get?: () => T;
+      set?: (value: T) => void;
+    }
+
+    declare export type ClassDecorator = <TFunction: Function>(target: TFunction) => TFunction;
```

Comment:
`Function` is `any` as a Fow 0.75, so the constraint here will not work. Maybe `() => void`?

Reply:
Fixed 031d068e304411e6f4d0473b1f82336e8a36b51c